internal/featureflags/provider.go

```diff
@@ -0,0 +1,72 @@
+package featureflags
+
+import (
+	"context"
+
+	"github.com/open-feature/go-sdk/openfeature"
+)
+
```

Comment:
Where is the interface? The provider being shipped here is a default, but openfga doesn't enforce any particular clients, thus an interface must be provided so that clients can be injected.